### ISSUE TITLE
Unknown memory leak

### ISSUE THREAD

**Issue by deliciouslytyped** [jesseduffield/lazydocker#492]
Throwback to #139, I'm on 0.20.0 and there is some sort of memory leak. Three lazydocker processes each ended up using 30% of RAM.
I don't now if it has to do with it, but I left the terminals in the "Press enter to return to lazydocker" screen after exiting shells.

**Comment by DovieW** [comment]
I'm noticing this as well on version `0.23.3`

I'm still diagnosing the issue, but I'm having dev machines crashing and I think it's because I'm leaving lazydocker running for long stretches.

I can see now one instance of lazydocker is using 1GB of memory

**Comment by MikaelElkiaer** [comment]
Also experiencing this - killing my 32G RAM machine simply by running overnight.
It's a big stack with lots of log output - which is why I suspect that log output is the reason for the high RAM usage.

**Comment by DovieW** [comment]
That makes sense, if lazydocker is keeping everything in memory. I have a container that has runaway logs sometimes accumlating multiple GBs. Maybe lazydocker can implement some kind of caching thing or something. Or just drop logs that go back a configurable number of lines maybe

**Comment by centralhardware** [comment]
> [2539473.683745] Out of memory: Killed process 815855 (lazydocker) total-vm:2548856kB, anon-rss:1877432kB, file-rss:0kB, shmem-rss:0kB, UID:0 pgtables:3780kB oom_score_adj:0

WTF

**Comment by kevincojean** [comment]
Yes, it's likely logging which blows up RAM usage; experienced this myself yesterday evening...

**Comment by h1kv** [comment]
This issue is still present on Lazydocker (0.25.2).

**Comment by etiedt** [comment]
Also experiencing this. Somehow a process was left running in the background and it wound up eating up 8G of memory over the course for 40k hours.

Ideally it wouldn't do this. Perhaps a stop gap could be that there is a config setting with a sane default that would cap the amount of memory it is allowed to use.

Version 0.25.0
